src/pane.coffee

```diff
@@ -39,18 +39,25 @@ class Pane extends Model
     @emitter = new Emitter
     @itemSubscriptions = new WeakMap
     @items = []
+    @itemStack = []
 
     @addItems(compact(params?.items ? []))
     @setActiveItem(@items[0]) unless @getActiveItem()?
+    @addItemsToStack(params?.itemStackIndices ? [])
     @setFlexScale(params?.flexScale ? 1)
 
   serialize: ->
     if typeof @activeItem?.getURI is 'function'
       activeItemURI = @activeItem.getURI()
+    itemsToBeSerialized = compact(@items.map((item) -> item if typeof item.serialize is 'function'))
+    itemStackIndices = []
+    for item in @itemStack
+      itemStackIndices.push(itemsToBeSerialized.indexOf(item)) if typeof item.serialize is 'function'
```

Comment:
So this looks a little awkward. Would be nice to use `compact`, but compact removes any falsy, so `0` items. A list comprehension works though: 

```
itemStackIndices = (itemsToBeSerialized.indexOf(item) for item in @itemStack when typeof item.serialize is 'function')
```